src/utils.js

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios';
+
+const baseUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+
+export async function createGame(user, score) {
+  const requestBody = {
+    user,
+    score,
+  };
+  await axios.post(`${baseUrl}5H3ZhzHBptAjLZzd7hXU/scores`, requestBody);
+}
+
+export async function getData() {
+  const response = await axios.get(`${baseUrl}5H3ZhzHBptAjLZzd7hXU/scores`);
+  return response.data;
+}
+
+export default {
+  createGame,
+  getData,
+};
```

Comment:
- [ ] Let's remove this export default, as we only have the need to export objects, functions, etc once per file. [Here is a useful link to remind ourselves of ES6 syntax.](https://devhints.io/es6#modules)

Reply:
Ok will check it out